paddle/operators/send_op.cc

```diff
@@ -19,59 +19,101 @@ limitations under the License. */
 #include "paddle/framework/lod_tensor.h"
 #include "paddle/framework/op_registry.h"
 
-#include "paddle/operators/detail/send_recv_impl.h"
-#include "paddle/operators/detail/simple_block_queue.h"
+#include <future>
+#include "paddle/operators/detail/grpc_client.h"
 
 namespace paddle {
 namespace operators {
 
-// TODO(typhoonzero): this is a simple implementation which only send
-// one tensor
+bool Send(const std::vector<std::string>& ep, const framework::Scope* scope,
+          const std::vector<std::string>& ins, int64_t time_out) {
+  detail::RPCClients c;
```

Comment:
Creating new client object in each send is inefficient.

Reply:
Done.Thanks.